projects/plugins/protect/src/js/state/reducers.js

```diff
@@ -11,6 +12,14 @@ import {
 	SET_NOTICE,
 } from './actions';
 
+const credentialState = ( state = {}, action ) => {
```

Comment:
Minor - I think `credentialState.state` is a little redundant, I'd recommend we either:
* A) Just call it `credentials`, so we can access it like `getCredentials()` and `credentials.state`. Good if we eventually need to add additional credentials properties aside from `state`.
```javascript
const { state } = useSelect( select => select( STORE_ID ).getCredentials() );
console.log( state ); // "active"
```
* B) Call it `credentialsState` and just use it as a string value. Keeps it a single, simple value while we only need that information.
```javascript
const credentialsState = useSelect( select => select( STORE_ID ).getCredentialsState() );
console.log( credentialsState ); // "active"
```

Reply:
I've been debating with myself about the naming on this. Technically this isn't really the state of credentials, but it's the rewind state and one of them can be awaiting_credentials which indicate that we still need to input credentials. Maybe we should rename this object from `credentialState` to `rewind` which then has state as a node to more accurately match the backend?

Reply:
Makes sense! There's option C as well, a `hasCredentials` boolean which we set based on the result of the rewind API state. Either works well for me I think!